tensorflow/core/grappler/optimizers/auto_mixed_precision.cc

```diff
@@ -1374,71 +1379,55 @@ bool AutoMixedPrecisionImpl::IsSourceOrSinkOp(const string& op) const {
 // Finds all clusters of float32 Tensor List nodes that are connected via their
 // handle edges. Unsafe clusters (those with edges that cross untraversable
 // boundaries via _Arg, _Ret, PartitionedCall etc. nodes) are added to black_set
-// and not returned.
+// and not returned. The caller should paint all nodes in a cluster the same
```

Comment:
Remove the "and not returned" part, as it is no longer true.

Reply:
Done